lib/vagrant-parallels/guest_cap/linux/mount_parallels_shared_folder.rb

```diff
@@ -79,11 +83,7 @@ def self.mount_parallels_shared_folder(machine, name, guestpath, options)
             sleep 2
           end
 
-          # Emit an upstart event if we can
-          if machine.communicate.test('test -x /sbin/initctl')
-            machine.communicate.sudo(
-              "/sbin/initctl emit --no-wait vagrant-mounted MOUNTPOINT=#{expanded_guest_path}")
-          end
+          emit_upstart_notification(machine, expanded_guest_path)
```

Comment:
`emit_upstart_notification` is available only in Vagrant 2.0+, so it will break the backward compatibility.
I'd recommend trying the old notation, as it was in `virtualbox` provider:

```ruby
        machine.communicate.sudo <<-EOH.gsub(/^ {10}/, "")
          if command -v /sbin/init && /sbin/init 2>/dev/null --version | grep upstart; then
            /sbin/initctl emit --no-wait vagrant-mounted MOUNTPOINT=#{guest_path}
          fi
        EOH
```
https://github.com/hashicorp/vagrant/commit/9f393fc1e0c1f0ceefb3f94b22f0bcb5b170371e#diff-97d643b35101562477f4b556a379415fL91

Reply:
Done as you suggest.